File: packages/hekireki/prisma/ecto/notification.ex
defmodule MyApp.Notification do
  use Ecto.Schema

  @primary_key {:id, :binary_id, autogenerate: true}
  schema "Notification" do
    field(:body, :string)
    field(:userId, :string)
    field(:createdAt, :naive_datetime)
    field(:user, :string)
  end
end

File: fixtures/twitter-clone-sample/sqlalchemy/notification.py
from sqlalchemy import DateTime, ForeignKey, String, func
from sqlalchemy.orm import Mapped, mapped_column, relationship
from datetime import datetime

from .base import Base


class Notification(Base):
    __tablename__ = "notification"

    id: Mapped[str] = mapped_column(String, primary_key=True)

    body: Mapped[str] = mapped_column(String, nullable=False)

    user_id: Mapped[str] = mapped_column(String, ForeignKey("user.id"), nullable=False)

    created_at: Mapped[datetime] = mapped_column(DateTime, nullable=False, server_default=func.now())

    user: Mapped["User"] = relationship("User", back_populates="notifications")
